scripts/seed_admins.py

```
@@ -0,0 +1,26 @@
+from django.contrib.auth import get_user_model
+
+
+def run():
```

Ruff (0.8.2)

Missing return type annotation for public function `run`

Add return type annotation: `None`

(ANN201)